File: CargoTransportApp.Services.CompanyAPI/DataAccess/Entities/ShippmentService.cs
using CargoTransportApp.Services.CompanyAPI.DataAccess.Entities.Base;

namespace CargoTransportApp.Services.CompanyAPI.DataAccess.Entities
{
	public class ShippmentService : BaseId
	{
		public int Parcel { get; set; }
		public int? MinimumDefaultDimension { get; set; }
		public int? MaximumDefaultDimension { get; set; }
		public int? MinimumDefaultWeight { get; set; }
		public int? MaximumDefaultWeight { get; set; }
		public double InitialDimensionPrice { get; set; }
		public double InitialWeightPrice { get; set; }
		public int CompanyId { get; set; }
		public Company Company { get; set; }
		public IEnumerable<DeliveryByDimension> DeliveryByDimensions { get; set; } = new List<DeliveryByDimension>();
		public IEnumerable<DeliveryByWeight> DeliveryByWeights { get; set; } = new List<DeliveryByWeight>();
	}
}

File: Frontend/CargoTransportClient/src/app/company/models/shippment-service.ts
import {DeliveryByDimension} from 'src/app/company/models/delivery-dimension';
import {DeliveryByWeight} from 'src/app/company/models/delivery-weight';
import { Company } from './company';
export interface ShippmentService{
  id:number,
  parcel:number,
  minimumDefaultDimension:number | null,
  maximumDefaultDimension:number | null,
  minimumDefaultWeight:number | null,
  maximumDefaultWeight:number | null,
  initialDimensionPrice:number,
  initialWeightPrice:number,
  companyId:number,
  company:Company,
  deliveryByDimensions:DeliveryByDimension[],
  deliveryByWeights:DeliveryByWeight[]
}
